<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>569</width>
    <height>383</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>569</width>
    <height>383</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>569</width>
    <height>383</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_turn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X's turn.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_vs_player">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VS Player</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_bigLogo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/images/large_logo.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>100</width>
      <height>92</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/small_logo_100px.png)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Select">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Score">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_X">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_O">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>O:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_O_score">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_X_score">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_new_game">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: 4px solid black;</string>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_signature">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Designed by Bachour &amp; Deleau</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_music">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_music">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_dark_mode">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Dark Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>681</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="place_0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="place_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>83</width>
      <height>83</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
background : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_win1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_vs_computer">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VS Computer</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_darkmode">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>label_turn</zorder>
   <zorder>RB_vs_player</zorder>
   <zorder>label_bigLogo</zorder>
   <zorder>label_logo</zorder>
   <zorder>label_Select</zorder>
   <zorder>label_Score</zorder>
   <zorder>label_X</zorder>
   <zorder>label_O</zorder>
   <zorder>label_O_score</zorder>
   <zorder>label_X_score</zorder>
   <zorder>PB_new_game</zorder>
   <zorder>label_signature</zorder>
   <zorder>label_music</zorder>
   <zorder>slider_music</zorder>
   <zorder>label_dark_mode</zorder>
   <zorder>place_0</zorder>
   <zorder>place_1</zorder>
   <zorder>place_2</zorder>
   <zorder>place_3</zorder>
   <zorder>place_4</zorder>
   <zorder>place_5</zorder>
   <zorder>place_6</zorder>
   <zorder>place_7</zorder>
   <zorder>place_8</zorder>
   <zorder>label_win1</zorder>
   <zorder>RB_vs_computer</zorder>
   <zorder>CB_darkmode</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
